cmake/Modules/opm-material-prereqs.cmake

```diff
@@ -16,8 +16,8 @@ set (opm-material_DEPS
 	# compile with C++0x/11 support if available
 	"CXX11Features REQUIRED"
 	# prerequisite OPM modules
-	"opm-common"
 	"opm-parser"
+	"opm-common"
```

Comment:
it is already REQUIRED because opm-material uses the opm build system. (thanks for the catch, BTW.)

having a way of building opm-common without depending on the libs would be nice, though...